build.sh

```diff
@@ -136,12 +142,38 @@ else
 	wait %1
 
 	# Run tests if activated
-	if [ $tests -eq 1 ]; then
+	if [ ${skip_tests} -eq 0 ] && [[ "${tests}" =~ .*"unittests".* ]]; then
 		echo "Running tests"
 		containerName=$(cat /proc/sys/kernel/random/uuid)
 		sudo podman run --name $containerName $dockerArgs --rm \
 			"${buildImage}" \
 			/opt/gardenlinux/bin/garden-test &
 		wait %1
 	fi
+	if [ ${skip_tests} -eq 0 ] && [[ "${tests}" == *"chroot"* ]]; then
+		echo "Creating config file for chroot tests"
+		containerName=$(cat /proc/sys/kernel/random/uuid)
+		prefix="$(${thisDir}/bin/garden-feat --featureDir $featureDir --features "$features" --ignore "$disablefeatures" cname)-$dpkgArch-$version-$commitid"
+		${thisDir}/bin/garden-integration-test-config chroot ${prefix} ${containerName} ${outputDir}
+		echo "Running pytests in chroot"
+		sudo podman run --cap-add SYS_ADMIN,MKNOD,AUDIT_WRITE,NET_RAW --security-opt apparmor=unconfined \
+			--name $containerName --rm -v `pwd`:/gardenlinux \
+			gardenlinux/integration-test:dev \
+			pytest --iaas=chroot --configfile=/gardenlinux/config/${containerName}.yaml &
+		wait %1
+		rm config/${containerName}.yaml
+	fi
+	if [ ${skip_tests} -eq 0 ] && [[ "${tests}" == *"kvm"* ]]; then
+		echo "Creating config file for KVM tests"
+		containerName=$(cat /proc/sys/kernel/random/uuid)
+		prefix="$(${thisDir}/bin/garden-feat --featureDir $featureDir --features "$features" --ignore "$disablefeatures" cname)-$dpkgArch-$version-$commitid"
+		${thisDir}/bin/garden-integration-test-config kvm ${prefix} ${containerName} ${outputDir}
+		echo "Running pytests in KVM"
+		sudo podman run --name $containerName --rm -v /boot/:/boot \
+			-v /lib/modules:/lib/modules -v `pwd`:/gardenlinux  \
+			gardenlinux/integration-test:dev \
```

Comment:
We should probably create a dedictaded image for this instead of using `gardenlinux/integration-test:dev` since it contains much more package than we need to perform our tests.

In addition, we should create a `container-test` target in the Makefile to ensure that this new test container (we may call it `gardenlinux/base-test`) already exists once we call build.sh